core/bones/base.py

```diff
@@ -444,7 +444,7 @@ def parseSubfieldsFromClient(self) -> bool:
     def singleValueFromClient(self, value, skel, name, origData):
         """
         Prevents the BaseBone from reading data using the fromClient method.
-        If needed, use the rawValueBone instead.
+        If needed, use the RawValueBone instead.
```

Comment:
The bone is called just `RawBone`.

```suggestion
        If needed, use the RawBone instead.
```